### PR TITLE
Fix: remove legacy minimun in base.py and replace it in elementwise.py

### PR DESCRIPTION
**Pull request by JLU-Neal**

### CODE REVIEW

File: ivy/container/elementwise.py [hunk 1]
```diff
@@ -6245,3 +6245,59 @@ def erf(
...
+    def static_minimum(
+        x1: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        x2: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
```
Comment (juliagsy) [inline ivy/container/elementwise.py:6259]: docstrings, same as `minimum` from below

File: ivy/container/elementwise.py [hunk 2]
```diff
@@ -6245,3 +6245,59 @@ def erf(
...
+            x2,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+            out=out,
+        )
+
+    def minimum(
+        self,
```
Comment (juliagsy) [inline ivy/container/elementwise.py:6272]: type hints for the arguments (same as `static_minimum` from above)

File: ivy/container/elementwise.py [hunk 3]
```diff
@@ -6245,3 +6245,59 @@ def erf(
...
+        to_apply=True,
+        prune_unapplied=False,
+        map_sequences=False,
+        out=None,
+    ):
+        """Computes the elementwise minimum between this container and another container
+        or number.
+
+        Parameters
+        ----------
```
Comment (juliagsy) [inline ivy/container/elementwise.py:6284]: missing self parameter, you can refer to functional `minimum` for description :)

File: ivy/container/elementwise.py [hunk 4]
```diff
@@ -6245,3 +6245,59 @@ def erf(
...
+        x2
+        The other container or number to compute the minimum against.
```
Comment (juliagsy) [inline ivy/container/elementwise.py:6286]: this is probably for consistency and readability, but missing indent for description, eg.

```
x2
     The other container or number to compute the minimum against.
```

File: ivy/container/elementwise.py [hunk 5]
```diff
@@ -6245,3 +6245,59 @@ def erf(
...
+        The key-chains to apply or not apply the method to. Default is None.
+        to_apply
+        If True, the method will be applied to key_chains, otherwise key_chains
+        will be skipped. Default is True.
+        prune_unapplied
+        Whether to prune key_chains for which the function was not applied.
+        Default is False.
+        map_sequences
+        Whether to also map method to sequences (lists, tuples). Default is False.
+
```
Comment (juliagsy) [inline ivy/container/elementwise.py:6297]: missing out parameter, you can refer to functional `minimum` for description :)

**Review by juliagsy — CHANGES_REQUESTED** [review]
just some docstrings additions or fix, otherwise looks good! (provided its test passes too!) 

thank you!

**Review by djl11 — CHANGES_REQUESTED** [review]
Julia's requests are correct - thanks Julia

**Comment by JLU-Neal** [comment]
> just some docstrings additions or fix, otherwise looks good!
> 
> thank you!
> 
> ..................
> 
> oh right! I forgot to mention one thing, could you also maybe check `test_container.py` to see if it has a test function for this legacy method? if yes, could you please remove it as well? since its tested in `test_minimum` in `test_elementwise` Thanks!

Hi Julia! Thank you! I have made changes according to your comments :D
And I have checked "test_container.py" and seems there is no related test function in there. So it should be fine!

**Pull request merged**